app/controllers/patients_controller.rb

```diff
@@ -363,6 +359,10 @@ def remove_from_household
       comment = "User removed monitoree from a household. Monitoree with ID #{old_hoh.id} will"\
                 ' no longer be responsible for handling their reporting.'
       History.monitoring_change(patient: current_patient, created_by: current_user.email, comment: comment)
+    else
+      # Create history item for old HoH
```

Comment:
This comment got separated from the code it applies to. Should be moved up into the `if` clause

Reply:
this should be addressed now